public/index.html

```diff
@@ -191,18 +194,7 @@ <h2>New Lesson</h2>
       <script src="https://www.gstatic.com/firebasejs/7.22.0/firebase-auth.js"></script>
       <script src="https://www.gstatic.com/firebasejs/7.22.0/firebase-firestore.js"></script>
       <script src="https://cdn.quilljs.com/1.3.6/quill.js"></script>
-      <script>
-        const firebaseConfig = {
-          apiKey: "AIzaSyBrrTgsW0PTZ2nPTs7mXcQrEyIbx9iSwqI",
-          authDomain: "dev-jot-9d423.firebaseapp.com",
-          databaseURL: "https://dev-jot-9d423.firebaseio.com",
-          projectId: "dev-jot-9d423",
-          storageBucket: "dev-jot-9d423.appspot.com",
-          messagingSenderId: "849978326803",
-          appId: "1:849978326803:web:8eeda6772f7fe24e14a4e6",
-          measurementId: "G-4B69RWW1GP",
-        };
-
+      <!-- <script>
         const toolbarOptions = [
```

Comment:
Instead of commenting this out we can now remove this 😄

Reply:
Oops, nice catch!